06_puppy/mohitnag/syncstore.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+)
+
+// CreatePuppy creates a Puppy in memstore
```

Comment:
The comments in syncstore should be updated (ie. they all say 'memstore')... I also saw a comment from Cam which suggested looking at the following link for some details on godoc comments (https://blog.golang.org/godoc-documenting-go-code)

Reply:
Please fix.

Reply:
Thanks for pointing out. Fixed